base/sort.jl

```diff
@@ -667,13 +667,15 @@ end
 
 Sort the vector `v` in place. [`QuickSort`](@ref) is used by default for numeric arrays while
 [`MergeSort`](@ref) is used for other arrays. You can specify an algorithm to use via the `alg`
-keyword (see [Sorting Algorithms](@ref) for available algorithms). The `by` keyword lets you provide
-a function that will be applied to each element before comparison; the `lt` keyword allows
-providing a custom "less than" function (note that for every `x` and `y`, only one of `lt(x,y)`
-and `lt(y,x)` can return `true`); use `rev=true` to reverse the sorting order. These
-options are independent and can be used together in all possible combinations: if both `by`
-and `lt` are specified, the `lt` function is applied to the result of the `by` function;
-`rev=true` reverses whatever ordering specified via the `by` and `lt` keywords.
+keyword (see [Sorting Algorithms](@ref) for available algorithms. Take into account the stability
+of the different sorting algorithms. QuickSort which is the default algorithm for numeric arrays
```

Comment:
Add tick marks for `QuickSort` and `MergeSort`? Xrefs to those? There should be a comma after `QuickSort` as well.